sql/mob_groups.sql

```diff
@@ -11608,7 +11608,7 @@ INSERT INTO `mob_groups` VALUES (13721,4736,81,0,0,2535,0,5000,90,92,0);
 INSERT INTO `mob_groups` VALUES (13722,4883,89,7200,0,0,8000,5000,70,72,0);
 INSERT INTO `mob_groups` VALUES (13723,4869,89,7200,0,4519,13000,0,80,80,0);
 INSERT INTO `mob_groups` VALUES (13724,4679,89,0,128,0,0,9999,90,95,0);
-INSERT INTO `mob_groups` VALUES (13725,4680,90,7200,128,4520,12000,9999,80,80,0);
+INSERT INTO `mob_groups` VALUES (13725,4680,90,7200,32,4520,12000,9999,80,80,0);
```

Comment:
if its placeholdered then respawn time here should be zero

Reply:
This is my first time ever doing any of this so I am very raw.  The person helping me create the mob said changing this from 128 to 32 signifies it's a lottery pop instead of a scripted pop.  Should the 32 be changed to a 0?

Reply:
no thats the spawn *type*, I'm talking the spawn *time*.

Reply:
Thanks, that change should also be live now assuming I pushed it properly.